apprise/plugins/NotifyTwilio.py

```diff
@@ -92,6 +92,8 @@ class NotifyTwilio(NotifyBase):
     templates = (
         '{schema}://{account_sid}:{auth_token}@{from_phone}',
         '{schema}://{account_sid}:{auth_token}@{from_phone}/{targets}',
+        '{schema}://{account_sid}:{api_key}:{auth_token}@{from_phone}',
+        '{schema}://{account_sid}:{api_key}:{auth_token}@{from_phone}/{targets}',
```

Comment:
This format doesn't seem ideal -- open to other suggestions!

Reply:
First off, great work.

But i'd keep it simple.  I wouldn't go changing the URL to much as it breaks it for other people.

It might just be easier to add the `{apikey}` as a keyword argument since it isn't really mandatory.

Your URLs maybe jsut need to be `twilio://legacy/url/?apkey=<new key>`

You can fiind lots of examples on how to do this.  Basically you'll just have a new entry to add to the `__init__()` for parsing it, `url()` for including it in the `params` and `parse_url()` to extract it (if present).

Thoughts?

Reply:
This seems like a better plan, thanks for the suggestion!